tests/lib/nsxv_test.py

```diff
@@ -367,9 +413,32 @@ def testTranslatePolicyWithEstablished(self):
                                                   nacaddr.IP('192.168.0.0/16')])
     self.naming.GetServiceByProto.return_value = ['123']
 
-    exp_info = 2
     pol = policy.ParsePolicy(INET_FILTER_WITH_ESTABLISHED, self.naming, False)
-    translate_pol = nsxv.Nsxv(pol, exp_info)
+    translate_pol = nsxv.Nsxv(pol, EXP_INFO)
+    nsxv_policies = translate_pol.nsxv_policies
+    for (_, filter_name, filter_list, terms) in nsxv_policies:
+      self.assertEqual(filter_name, 'inet')
+      self.assertEqual(filter_list, ['inet'])
+      self.assertEqual(len(terms), 1)
+
+      self.assertNotIn('<sourcePort>123</sourcePort><destinationPort>123',
+                       str(terms[0]))
+
+    self.naming.GetServiceByProto.assert_has_calls(
+        [mock.call('NTP', 'udp')] * 2)
+
+  def testTranslatePolicyWithEstablished(self):
```

Comment:
Looks like this got added again when trying to resolve merge conflict. Removed this one.